test/e2e/upgrade/common.go

```diff
@@ -94,6 +95,12 @@ func createBackupAndWaitToSucceed(name string, t *testing.T) *v2.Backup {
 		},
 		Spec: v2.BackupSpec{
 			Cluster: name,
+			Resources: &v2.BackupResources{
+				Caches:       []string{volatileCacheName},
```

Comment:
This can be simplified to:

```go
		Spec: v2.BackupSpec{
			Cluster: name,
			Resources: &v2.BackupResources{
				Caches: []string{volatileCacheName},
			},
		},
```

Reply:
Yeah my idea was to explicitly pass empty lists to be 100% sure about the fact to not import other resources... Anyway, I applied the suggestion. Thanks

Reply:
I don't think you did :stuck_out_tongue_winking_eye: